charts/virtual-kubelet-0.1.1-rbac/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,14 @@
+{{ if .Values.rbac.install }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "fullname" . }}-role-binding
+subjects:
+- kind: ServiceAccount
+  name: default
```

Comment:
shall we create a service account for VK? {{ template "fullname" }}-service-account?

Reply:
Implemented